python/pipeline/experiment.py

```diff
@@ -170,6 +172,60 @@ class Software(dj.Lookup):
         ('imager', '1.0')]
 
 @schema
+class LaserCalibration(dj.Manual):
+    definition = """
+    # stores measured values from the laser power calibration
+
+    calibration_id      : int
+    ---
+    -> Person
```

Comment:
Because it handles all the calibration that could influence the measurement (e.g. the Pockels cell).